un_sdg/main.py

```diff
@@ -0,0 +1,26 @@
+"""executes bulk dataset import + chart updates for the UN_SDGs dataset.
+Usage:
+    python -m un_sdg.main
+"""
+from un_sdg import DATASET_DIR, DATASET_VERSION
+
+from un_sdg import (
+    download, 
+    clean,
+    match_variables
+)
+
+from standard_importer import import_dataset
+from standard_importer.chart_revision_suggester import ChartRevisionSuggester
+
+def main():
+    download.main()
+    clean.main()
+    import_dataset.main(DATASET_DIR, DATASET_VERSION)
+    match_variables.main()
+
+    suggester = ChartRevisionSuggester('un_sdg')
+    suggester.suggest()
```

Comment:
From skimming a few of the suggested chart revisions in the chart approval UI, most of the suggested revisions look good. 

But there are at least a few where the data looks wonky in the suggested chart revision. e.g. chart Ids 2047, 2048, 2051. Is there an issue with the data in these revised charts? Or is it a problem with the `ChartRevisionSuggester`?